include/h2o.h

```diff
@@ -439,6 +447,11 @@ struct st_h2o_globalconf_t {
         h2o_socketpool_t global_socketpool;
     } proxy;
 
+    /**
+     * enum indicating that h2o forward 1xx response from upstream
+     */
+    h2o_forward_informational_t forward_informational;
```

Comment:
The doc-comment says the enum indicates if 1xx _from upstream_ is forwarded.

Is that true?

My understanding is that this function controls if protocol handlers sends 1xx responses. In other words, this property controls what h2o sends rather than what it forwards (consider the hypothetical case of a file handler emitting 1xx; it's not "upstream", since in such case h2o is the origin).

Considering that, I'd prefer renaming this from "forward_informational_mode" to "send_informational_mode" (see my other comment about appending "_mode").